internal/cli/deployments/setup.go

```diff
@@ -598,7 +598,10 @@ func (opts *SetupOpts) runLocal(ctx context.Context) error {
 	}
 
 	if err := opts.createLocalDeployment(ctx); err != nil {
-		_ = opts.RemoveLocal(ctx)
+		// in case the deployment already exists we shouldn't delete it
+		if !strings.Contains(err.Error(), "deployment already exists and is currently in") {
```

Comment:
I wonder if we there is a better error checking approach? like `errors.Is`?

Reply:
I was wondering the same thing. But the error contains some dynamic parts, hence the contains.

Reply:
Updated using your suggestion

Reply:
was late with my comment :)